tillerinobot/src/main/java/tillerino/tillerinobot/lang/Indonesian.java

```diff
@@ -0,0 +1,271 @@
+package tillerino.tillerinobot.lang;
+
+import java.util.List;
+import java.util.Random;
+
+import org.tillerino.osuApiModel.Mods;
+import org.tillerino.osuApiModel.OsuApiUser;
+
+import tillerino.tillerinobot.CommandHandler.Action;
+import tillerino.tillerinobot.CommandHandler.Message;
+import tillerino.tillerinobot.CommandHandler.Response;
+
+/**
+ * TRANSLATION NOTE:
+ * 
+ * Please put some contact data into the following tag. If any additional
+ * messages are required, I'll use the English version in all translations and
+ * notify the authors.
+ * 
+ * @author "[ RumberkrenZ ]" rrr.ee.uu.bb.ee.nnn@gmail.com https://github.com/Rumberkren https://osu.ppy.sh/u/3925053
+ */
+public class Default implements Language {
+	static final Random rnd = new Random();
+
+	@Override
+	public String unknownBeatmap() {
+		return "Maaf, saya tidak mengetahui map itu. Mungkin itu adalah map yang baru, terlalu sulit, tidak di rank atau bukan mode standard osu.";
+	}
+
+	@Override
+	public String internalException(String marker) {
+		return "Umm... Sepertinya Tillerino asli mengacaukan pengaturan saya."
+				+ "Jika dia tidak segera mengetahuinya, bisakah kau [https://github.com/Tillerino/Tillerinobot/wiki/Contact memberitahunya]? (reference "
+				+ marker + ")";
+	}
+
+	@Override
+	public String externalException(String marker) {
+		return "A[a yang terjadi? Saya hanya mendapatkan omong kosong dari server osu. Bisakah kau memberitahu saya apa yang terjadi? 0011101001010000"
+				+ "Tillerino asli mengatakan bahwa ini tidak perlu dikhawatirkan, dan kita harus mencoba lagi."
+				+ "Jika kau mencemaskan ini, kau bisa [https://github.com/Tillerino/Tillerinobot/wiki/Contact memberitahunya] tentang ini. (reference "
+				+ marker + ")";
+	}
+
+	@Override
+	public String noInformationForModsShort() {
+		return "Tidak ada data untuk mod yang diminta.";
+	}
+
+	@Override
+	public Response welcomeUser(OsuApiUser apiUser, long inactiveTime) {
+		if (inactiveTime < 60 * 1000) {
+			return new Message("beep boop");
+		} else if (inactiveTime < 24 * 60 * 60 * 1000) {
+			return new Message("Selamat datang kembali, " + apiUser.getUserName() + ".");
+		} else if (inactiveTime > 7l * 24 * 60 * 60 * 1000) {
+			return new Message(apiUser.getUserName() + "...")
+				.then(new Message("...apakah itu kau? Sudah lama sekali!"))
+				.then(new Message("Senang bisa menemuimu kembali. Bisakah saya menarik minat anda dalam sebuah rekomendasi?"));
+		} else {
+			String[] messages = {
+					"Kau terlihat seperti kau ingin sebuah rekomendasi.",
+					"Sangat senang bisa menemuimu! :)",
+					"Manusia favoritku. (Jangan beritahu manusia lain!)",
+					"Kejutan yang ramah! ^.^",
+					"Saya berharap kau akan datang. Semua manusia lain bodoh, tapi jangan beritahu yang lain bahwa saya berkata seperti itu! :3",
+					"Apa yang anda rasa ingin anda lakukan hari ini?",
```

Comment:
all the messages in this list above are not a start of a sentence, so they probably shouldnt be with an uppercase